packages/contracts/hardhat.config.ts

```diff
@@ -147,17 +156,17 @@ const config: HardhatUserConfig = {
 
   namedAccounts: {
     deployer: 0,
-    alice: 0,
-    bob: 1,
-    carol: 2,
-    dave: 3,
-    eve: 4,
-    frank: 5,
-    grace: 6,
-    harold: 7,
-    ivan: 8,
-    judy: 9,
-    mallory: 10,
+    alice: 1,
```

Comment:
I would assume you would be better re-using the named accounts defined earlier in the file than defining twice

Reply:
Fixed